DeepSelectNet/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Tensorflow 2.2 requires CUDA 10.1 and cuDNN 7.4
+FROM nvcr.io/nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04 AS build
+
+WORKDIR /app
+
+RUN apt-get update
+RUN apt-get install -y --no-install-recommends build-essential gcc wget zlib1g-dev
+
+# DeepSelectNet requires Python 3.5 >= version <= 3.8
+# Download and install Python 3.8
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get install -y libffi-dev libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev
```

Comment:
after each apt-get install (ideally in one RUN) = add apt-get clean and rm -rf /var/lib/apt/lists/* to reduce image size

Reply:
also lets combine all apt-get install in one RUN